tests/tests.c

```diff
@@ -454,7 +454,7 @@ bool sirtest_filecachesanity(void) {
     if (pass)
         print_expected_error();
 
-    sir_info("test test test");
+    (void)sir_info("test test test");
```

Comment:
this should be a _sir_eqland(pass, sir_info())

Reply:
Ah, you got it there before I saw this, thanks.